mantidproject/mantid
# Periods crash in muon analysis

Muon analysis was crashing when multiple period data was loaded into the single fit. 

**To test:**


The following instructions caused the error 'Fit Algorithm failed. Unknown property search object InputWorkspace_1'

1. Open the Muon Analysis interface
2. Load MUSR00015189.nxs 
3. Switch between the options in Group/Group Pair combo box on the home tab (two is enough).
4. Go to Settings tab, make sure multiple fitting is disabled
5. Go to Data Analysis tab
6. Add a function (e.g. flatBackground) and do a fit. It should work.
7. Change the workspace and do another fit. These should also (now) work. 

Ignore the message "cannot reach the specified tolerance in F", this just means that the fit was poor. 

Fixes #19975. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
